docs/changelog/81237.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 81237
+summary: More avoidance for `loadAuthorizedIndices`
```

Comment:
Can we update this with a more user-friendly description?

Reply:
I changed it to:
> Avoid loading authorized indices when requested indices are all concrete names